dolphinscheduler-ui-next/src/service/service.ts

```diff
@@ -22,6 +22,8 @@ import axios, {
   AxiosRequestHeaders,
 } from 'axios'
 import qs from 'qs'
+import _ from 'lodash'
+import $ from 'jquery'
```

Comment:
To download a file, you can get the corresponding `url` from the corresponding interception in `axios`, and then create a form for downloading. It is not recommended to use `jquery` for this.